sdk/storage/storage-file-share/test/node/highlevel.node.spec.ts

```diff
@@ -436,34 +437,49 @@ describe("Highlevel Node.js only", () => {
 
   it("fileClient.download should download data failed when exceeding max stream retry requests", async () => {
     recorder.skip("node", "Temp file - recorder doesn't support saving the file");
-    await fileClient.uploadFile(tempFileSmall, {
-      rangeSize: 4 * 1024 * 1024,
-      concurrency: 20
-    });
+    for (let index = 0; index < 1000; index++) {
+      await fileClient.uploadFile(tempFileSmall, {
+        rangeSize: 4 * 1024 * 1024,
+        concurrency: 20
+      });
 
-    const downloadedFile = path.join(tempFolderPath, recorder.getUniqueName("downloadfile."));
+      const downloadedFile = path.join(tempFolderPath, recorder.getUniqueName("downloadfile."));
 
-    let retirableReadableStreamOptions: RetriableReadableStreamOptions;
-    let injectedErrors = 0;
-    let expectedError = false;
+      let retirableReadableStreamOptions: RetriableReadableStreamOptions;
+      let injectedErrors = 0;
+      let expectedError = false;
 
-    try {
-      const downloadResponse = await fileClient.download(0, undefined, {
-        maxRetryRequests: 0,
-        onProgress: () => {
-          if (injectedErrors++ < 1) {
-            retirableReadableStreamOptions.doInjectErrorOnce = true;
+      try {
+        const downloadResponse = await fileClient.download(0, undefined, {
+          maxRetryRequests: 0,
+          onProgress: () => {
+            if (injectedErrors++ < 1) {
+              retirableReadableStreamOptions.doInjectErrorOnce = true;
+            }
           }
+        });
+        retirableReadableStreamOptions = (downloadResponse.readableStreamBody! as any).options;
+        await readStreamToLocalFileWithLogs(downloadResponse.readableStreamBody!, downloadedFile);
+      } catch (error) {
+        expectedError = true;
+      }
+
+      assert.ok(expectedError);
+      // fs.unlinkSync(downloadedFile);
+      let unlink_count = 0;
+      let successfully_unlinked = false;
+      do {
+        try {
+          unlink_count++;
+          console.log("iteration - ", index, "unlink_count - ", unlink_count);
+          fs.unlinkSync(downloadedFile);
+          successfully_unlinked = true;
+        } catch (error) {
+          console.log(error.message);
+          await delay(10);
         }
-      });
-      retirableReadableStreamOptions = (downloadResponse.readableStreamBody! as any).options;
-      await readStreamToLocalFile(downloadResponse.readableStreamBody!, downloadedFile);
-    } catch (error) {
-      expectedError = true;
+      } while (!successfully_unlinked);
     }
-
-    assert.ok(expectedError);
-    fs.unlinkSync(downloadedFile);
   });
```

Comment:
**[Temporary]** In case you want to bypass the timeout, do this, @jeremymeng .

```suggestion
  }).timeout(100000000000);
```

With this, the timeout shouldn't be a problem locally. 
CI might throw an error later on if the job takes too much time.